src/doc/unstable-book/src/rvalue-static-promotion.md

```diff
@@ -0,0 +1,5 @@
+# `rvalue_static_promotion`
+
+The tracking issue for this feature is: [#38865]
+
+------------------------
```

Comment:
Not a deal-breaker, but it'd be great if there was a sentence and/or example here briefly describing the feature. There are some examples here in [this PR](https://github.com/rust-lang/rust/pull/40452/files